helm-github-stars.el

```diff
@@ -178,26 +208,46 @@ When disabled (nil) don't align description."
   (let ((stars-list [])
         (repos-list [])
         (cache-hash-table (make-hash-table :test 'equal)))
-    ;; Fetch user's starred repositories
-    (let ((next-request t)
-          (current-page 1))
-      (while next-request
-        (let ((response (hgs/parse-github-response (hgs/request-github-stars current-page))))
-          (if (= 0 (length response))
-              (setq next-request nil)
-            (progn
-              (setq stars-list (vconcat stars-list response))
-              (setq current-page (1+ current-page)))))))
-    ;; Fetch user's repositories
-    (let ((next-request t)
-          (current-page 1))
-      (while next-request
-        (let ((response (hgs/parse-github-response (hgs/request-github-repos current-page))))
-          (if (= 0 (length response))
-              (setq next-request nil)
-            (progn
-              (setq repos-list (vconcat repos-list response))
-              (setq current-page (1+ current-page)))))))
+    (if helm-github-stars-token
```

Comment:
There is a mistake here! When `helm-github-stars-token` is provided, only first github page is fetched, so repository list is incomplete. I'm looking how to improve this..